week_1/.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    "python.defaultInterpreterPath": "C:\\Users\\SESA499711\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\corise-dagster-ill-k1v9-py3.7",
+    "python.testing.pytestEnabled" : true
+}
```

Comment:
Not an official reviewer, but I think it would make sense to create a .gitignore to avoid commiting these type of files. 

If you haven't ever used one, you could also use a generator like https://gitignore.io/ and create a .gitignore file in the root folder of the project. If you then try to commit something, it will avoid committing anything that's in that file, useful as well for passwords or specific things to your environment.

Reply:
I forgot to add vscode in gitignore. I will do the needful.